lib/artist.rb

```diff
@@ -0,0 +1,50 @@
+class Artist
+
+  attr_accessor :name, :song, :musiclibrarycontroller, :musicimporter
```

Comment:
It doesn't look like you are using most of these `attr_accessor`s. Only use what you need.

Reply:
I got rid of `:musiclibrarycontroller`, `:musicimporter` entirely. 
I also moved `:song` over as a reader instead.